Marcio/registrar revamp (#2052)
cloudflare/cloudflare-docs: products/registrar/src/content/why-choose-cloudflare/whois-redaction.md

---
order:
pcx-content: reference
---

# WHOIS redaction

Cloudflare Registrar provides personal data redaction on WHOIS information, if permitted by the Registry.

WHOIS is a standard for publishing the contact and nameserver information for all registered domains. Each registrar maintains their own WHOIS service. Anyone can query the registrar’s WHOIS service to reveal the data behind a given domain.

> However, broadcasting the registrant contact information via the WHOIS service can invite mountains of spam to your personal addresses. Cloudflare Registrar offers personal data redaction on WHOIS for free, that meets current ICANN guidelines.

Reviewer: ```suggestion
However, broadcasting the registrant contact information via the WHOIS service can cause spam mail to be delivered to your personal addresses. Cloudflare Registrar offers personal data redaction on WHOIS for free, that meets current ICANN guidelines.
```

Cloudflare’s WHOIS service can be found at https://rdap.cloudflare.com/. Select **WHOIS** as the search type.

## What is WHOIS redaction?

WHOIS redaction removes most contact information categorized as personal data (registrant name, email address, postal address) from the published WHOIS record for a domain. Fields will read “Data Redacted”. The nameserver, domain lock information, and date records for a domain are still available publicly. The following fields will continue to show in WHOIS: 

* Registrant State/Province 
* Registrant Country.

Cloudflare still maintains the authoritative, unredacted, record of your WHOIS data. You can modify this information at any time in. To access it:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account / domain.
1. Click **Registrar**.

> 1. Under the Manage tab, click **Edit** to update your information.

Reviewer: Are these accurate instructions? I don't see **Registrar** as an option after selecting a site. Where is **Registrar**?

Author: Upper-right corner of the interface. I've avoided writing directions like this as the interface might change and then it doesn't make sense to say "upper-right corner". 
<img width="1235" alt="Screen Shot 2021-08-18 at 14 43 12" src="https://user-images.githubusercontent.com/83226960/129908925-1e1098a1-601a-42f8-a75c-976430c9a463.png">

Reviewer: Oh I see. Then I think these instructions should change to 

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login).
2. In your **Account Home**, go to **Registrar**.

Because if I selected a domain, it would take me to the dash and I got lost following the instructions.

Author: Done

<Aside type="note">

WHOIS redaction is not the same as WHOIS privacy. WHOIS privacy replaces your information with proxy contact information, while redaction removes it altogether.

</Aside>

> Some registries do not allow domains to be registered with WHOIS Redaction or WHOIS Privacy. Many of these are country TLDs (ccTLDs), like `.uk.` Cloudflare does not yet support TLDs that prohibit WHOIS redaction, but we will in the near future.

Reviewer: ```suggestion
Some registries do not allow domains to be registered with WHOIS Redaction or WHOIS Privacy. Many of these are country TLDs (ccTLDs), like `.uk.` Cloudflare does not yet support TLDs that prohibit WHOIS redaction, but will in the near future.
```
